src/ngraph/runtime/cpu/builder/slice.cpp

```diff
@@ -48,52 +48,64 @@ namespace ngraph
                 auto lower_bounds = slice->get_lower_bounds();
                 auto upper_bounds = slice->get_upper_bounds();
 
-                bool strided = false;
-                for (auto stride : strides)
+                if (runtime::cpu::mkldnn_utils::use_mkldnn_kernel(node))
                 {
-                    if (stride != 1)
-                    {
-                        strided = true;
-                        break;
-                    }
-                }
+                    auto& mkldnn_emitter = external_function->get_mkldnn_emitter();
+                    auto input_desc = mkldnn_utils::get_input_mkldnn_md(node, 0);
+                    auto result_desc = mkldnn_utils::get_output_mkldnn_md(node, 0);
 
-                if (strided)
-                {
-                    std::function<decltype(runtime::cpu::kernel::strided_slice<float, 2>)> kernel;
+                    auto slice_index = mkldnn_emitter->build_slice(
```

Comment:
Is the mkldnn primitive actually faster than the existing kernel? Or is the primary motivation to eliminate layout conversions?

Reply:
to eliminate layout conversions.